pkg/engine/engine.go

```diff
@@ -176,12 +176,12 @@ func tplFun(parent *template.Template, includedNames map[string]int, strict bool
 		// text string. (Maybe we could use a hash appended to the name?)
 		t, err = t.New(parent.Name()).Parse(tpl)
 		if err != nil {
-			return "", errors.Wrapf(err, "cannot parse template %q", tpl)
+			return "", errors.Wrapf(err, "Cannot parse template %q", tpl)
```

Comment:
Isn't goonic/goish/golike/goey to start errors with a lower case letter?

Reply:
It might be I can switch it. Right now I am trying to figure out why my build is failing.

Reply:
The test is failing because of this change `Requested version %q does not exist for plugin %q` should be lower case "requested"

Reply:
fixed this